packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -3222,6 +3216,27 @@ function remountFiber(
   }
 }
 
+function checkScheduledUpdateOrContext(
+  current: Fiber,
+  renderLanes: Lanes,
+): boolean {
+  // Before performing an early bailout, we must check if there are pending
+  // updates or context.
+  const updateLanes = current.lanes;
+  if (includesSomeLane(renderLanes, updateLanes)) {
```

Comment:
`current` is first in the function definition, but second in this invocation. snip snap, snap snip

Reply:
Yeah based on my "subject-object" rationale this should be flipped